test/locators.spec.ts

```diff
@@ -27,11 +27,13 @@ describe('Given a practice form page', () => {
           'Switch Commands',
           'Wait Commands',
           'WebElement Commands'],
+        picture: './resources/calcifer.jpg',
       });
-      await personalInformationPage.submitForm();
     });
 
-    it('Then should have a title', async () => {
+    it('Then the image should have been uploaded and should have a title', async () => {
+      expect(await personalInformationPage.getProfilePictureValue()).toEqual('calcifer.jpg');
```

Comment:
I don't see the calcifieer.jpg file as part of this PR
![image](https://user-images.githubusercontent.com/1211491/124610549-3abd0b00-de36-11eb-8bc6-b338a68c34a1.png)

Reply:
![image](https://user-images.githubusercontent.com/39042313/124635466-19b3e480-de4d-11eb-9a81-2c1bcbf32aa1.png)